MatrixKit/Models/Contact/MXKContact.h

```diff
@@ -82,6 +83,17 @@ extern NSString *const kMXKContactDefaultContactPrefixId;
 @property (nonatomic, readonly) NSArray* matrixIdentifiers;
 
 /**
+ The matrix avatar url used (if any) to build the current thumbnail, nil by default.
+ */
+@property (nonatomic, readonly) NSString* matrixAvatarURL;
+
+/**
+ Reset the current thumbnail if it is retrieved from a matrix url. May be used in case of the matrix avatar url change.
+ A new thumbnail will be automatically restored from the contact data.
+ */
+- (void)resetMatrixThumbnail;
```

Comment:
For me there is a slight difference: we use thumbnail to talk about the contact picture. This thumbnail may be the local contact picture or an image retrieved from the matrix avatar url with a specific size...